intel_extension_for_transformers/transformers/modeling/kv_cache_compression/prune/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
```

Comment:
better move these files to other places, they should not be in the modeling folder